Sources/StreamChatUI/Composer/ComposerVC.swift

```diff
@@ -382,7 +382,7 @@ open class ComposerVC: _ViewController,
         }
 
         if !content.isEmpty && channelConfig?.typingEventsEnabled == true {
-            channelController?.sendKeystrokeEvent()
+            channelController?.sendKeystrokeEvent(parentMessageId: content.threadMessage?.id)
```

Comment:
we don't need to check config in the UI level, we do that inside `sendKeystrokeEvent`